logstash-core/src/main/java/org/logstash/ext/JrubyMemoryReadBatchExt.java

```diff
@@ -11,20 +11,32 @@
 import org.jruby.runtime.Block;
 import org.jruby.runtime.ThreadContext;
 import org.jruby.runtime.builtin.IRubyObject;
+import org.logstash.RubyUtil;
 
 @JRubyClass(name = "MemoryReadBatch")
 public class JrubyMemoryReadBatchExt extends RubyObject {
 
     private LinkedHashSet<IRubyObject> events;
 
     public JrubyMemoryReadBatchExt(final Ruby runtime, final RubyClass metaClass) {
```

Comment:
Had to keep this constructor due to the static setup performed in the `RubyUtil` class.

Reply:
@danhermann Right ... my bad :)